### DISCUSSION TITLE
How to visualize the concentration found in the tortuosity in a 3d medium?

### DISCUSSION THREAD

**Discussion by josephinoo** [PMEAL/porespy#596] [Q&A] [answered]
Hello, I am an undergraduate student, I want to be able to visualize in 3d the concentration obtained by the tortuosity

**Comment by jgostick** [comment, marked as answer]
Hi @eljosephavila123, the ``tortuosity`` function returns the concentration field as an image, with each pixel/voxel containing its concentration.  This will be in the form of a numpy array.  You can save this array as a tiff image using ``imageio``.  You can then open this up in paraview.  The colors should follow the values in the original image, so you can see concentration profiles.  However, this won't look very interesting since diffusion is just a smooth and steady gradient.  The main point of the ``tortuosity`` function is just to get the tortuosity value.
